packages/zk-certificates/contracts/GalacticaTwitterSBT.sol

```diff
@@ -128,6 +152,24 @@ contract GalacticaTwitterSBT is ERC721, AccessControl {
         return baseURI;
     }
 
+    function name(uint256 tokenId) public view returns(string memory) {
```

Comment:
This way to customize the name and symbol for individual token IDs is not part of the ERC721 standard for metadata. https://eips.ethereum.org/EIPS/eip-721
The ERC721 metadata extension only has a custom URI for individual tokens. The name and symbol are the same.
Is there a good reason for not going with the standard? I think the metadata.json offers enough customizability.

Reply:
I agree, it was intended to be a subname.